### ISSUE TITLE
Seperating Scroll Views

### ISSUE THREAD

**Issue by chid** [segment-boneyard/socrates#5]
Hey I would like to have separable scrolls since often the page goes off the screen with my markdown (120 lines)! 

I have tried to use http://www.waynemoir.com/wp-content/uploads/2010/12/split.page.with.no.frames.html 

however it seems like that typing in the textarea resets the position for some reason.

**Comment by SimplGy** [comment]
Yeah. I've seen the same issue. If you make a long document, the input text is very separate from the output.

Might make sense to match the height of the input text by line, or by paragraph

**Comment by dsernst** [comment]
Came to suggest the same. Socrates is awesome but doesn't look like it's being maintained anymore.
